scratch/copy-mk.py

```diff
@@ -43,15 +43,20 @@
 
 import katgpucbf.configure_tools
 
+#: Scale factors between MK and MK+ wideband gains, based on number of channels
+GAIN_SCALE = {
+    1024: 5687.284403271646,
+    4096: 3096.9068735303676,
+    32768: 1147.5800238145268,
+}
+
```

Comment:
Is this from any particular file/source of truth?

Reply:
It's computed from https://docs.google.com/document/d/1TZWZATFNrwVG3ugBEe2aaQVnJDY_plfu8QSgN5sfGsk/edit?tab=t.0#heading=h.x68achryfapa - but based on Ludwig's preliminary analysis, the values might be wrong. I'm tracking things in NGC-1636.